components/brave_ads/core/internal/ml/transformation/normalization_transformation_unittest.cc

```diff
@@ -42,15 +42,15 @@ TEST_F(BatAdsNormalizationTransformationTest, NormalizationTest) {
 
   std::vector<double> components;
   double s = 0.0;
-  for (const double x : norm_data->GetValuesForTesting()) {
-    components.push_back(x);
-    s += x * x;
+  for (const double component : norm_data->GetData()) {
```

Comment:
would it make more sense to name `norm_data` as `vector_data` to match the object name?